package.json

```diff
@@ -30,6 +30,7 @@
     "prom-client": "^12.0.0",
     "prop-types": "^15.8.1",
     "react": "17.0.2",
+    "react-confirm-alert": "^2.8.0",
```

Comment:
This dependency doesn't look promising based off use/issues/PRs https://github.com/GA-MO/react-confirm-alert
I'd flag this for a real app but is fine for prototypes